File: RSPAClient/Core/AtPar.POCOEntities/MT_ATPAR_PATIENT_CACHE.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_ATPAR_PATIENT_CACHE
    {
        public string ORG_ID { get; set; }
        public string PATIENT_MRC { get; set; }
        public string PATIENT_ACCNUMBER { get; set; }
        public string PATIENT_NAME { get; set; }
        public string PATIENT_BEDNUMBER { get; set; }
        public string PATIENT_DEPARTMENT { get; set; }
        public string PATIENT_SEX { get; set; }
        public string PATIENT_CLASS { get; set; }
        public System.DateTime MESSAGE_DATETIME { get; set; }
        public Nullable<System.DateTime> PATIENT_ADMIT_DATE { get; set; }
        public Nullable<System.DateTime> PATIENT_DISCHARGE_DATE { get; set; }
        public string PATIENT_VISIT_NUMBER { get; set; }
        public string STATUS { get; set; }
        public string OLD_PATIENT_MRC { get; set; }
        public string ITEM_ID { get; set; }
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/MT_ATPAR_PATIENT_CACHE.ts
export class MT_ATPAR_PATIENT_CACHE {
    public ORG_ID: string;
    public PATIENT_MRC: string;
    public PATIENT_ACCNUMBER: string;
    public PATIENT_NAME: string;
    public PATIENT_BEDNUMBER: string;
    public PATIENT_DEPARTMENT: string;
    public PATIENT_SEX: string;
    public PATIENT_CLASS: string;
    public MESSAGE_DATETIME:  Date;
    public PATIENT_ADMIT_DATE?: Date;
    public PATIENT_DISCHARGE_DATE?: Date;
    public PATIENT_VISIT_NUMBER: string;
    public STATUS: string;
    public OLD_PATIENT_MRC: string;
    public RBFlAG: boolean = false; 
    public ITEM_ID: string;
}